grafana/configmaps/datasource.yaml

```diff
@@ -15,6 +15,13 @@ data:
       user: $(OVERDUEFORECAST_USER)
       secureJsonData:
         password: $(OVERDUEFORECAST_PASSWORD)
+    - name: Postgres-overbooking
+      type: postgres
+      url: $(OVERBOOKING_POSTGRESS_URL)
+      database: overbooking
+      user: $(OVERBOOOKING_USER)
+      secureJsonData:
+        password: $(OVERBOOKING_PASSWORD)
       jsonData:
```

Comment:
The `jsonData` is part of each postgres datasource connection so you have to keep it for overdue connection and also for overbooking